internal/server/volume/server.go

```diff
@@ -221,38 +239,52 @@ func (s *Server) GetVolumeDiskNumber(context context.Context, request *internal.
 		return nil, fmt.Errorf("volume id empty")
 	}
 
-	diskNumber, err := s.hostAPI.GetVolumeDiskNumber(volumeId)
+	diskNumber, err := s.hostAPI.GetDiskNumberFromVolumeID(volumeId)
 	if err != nil {
 		klog.Errorf("failed GetVolumeDiskNumber %v", err)
 		return nil, err
 	}
 
-	response := &internal.VolumeDiskNumberResponse{
+	response := &internal.GetDiskNumberFromVolumeIDResponse{
 		DiskNumber: diskNumber,
 	}
 
 	return response, nil
 }
 
 func (s *Server) GetVolumeIDFromMount(context context.Context, request *internal.VolumeIDFromMountRequest, version apiversion.Version) (*internal.VolumeIDFromMountResponse, error) {
-	klog.V(4).Infof("calling GetVolumeFromMount with request %+v", request)
+	getVolumeIDFromTargetPathRequest := &internal.GetVolumeIDFromTargetPathRequest{
```

Comment:
I think we could add this check however a newer client v1beta3 will be unable to reach this endpoint because it's not declared in its proto file, the same thing happens with an pre v1beta3 client that wants to reach some of the new v1beta3 endpoints, they would be unreachable too